src/js/grapl-cdk/lib/services/model_plugin_deployer.ts

```diff
@@ -55,7 +55,7 @@ export class ModelPluginDeployer extends cdk.NestedStack {
             ),
             vpc: props.vpc,
             environment: {
-                GRAPL_LOG_LEVEL: props.defaultLogLevel,
+                GRAPL_LOG_LEVEL: props.logLevels.defaultLogLevel,
```

Comment:
Should we have a specific log level for this service?

Reply:
i mean probably but that wasnt the original intent of this PR lol